INSTALL.md

```diff
@@ -101,7 +101,7 @@ These commands should be run from a local optimade-python-tools directory.
 The following command starts a local Elasticsearch v6 instance, runs the test suite, then stops and deletes the containers (required as the tests insert some data):
 
 ```shell
-docker run -d --name elasticsearch_test -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -e "xpack.security.enabled=false" elasticsearch:7.17.1 \
+docker run -d --name elasticsearch_test -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -e "xpack.security.enabled=false" elasticsearch:7.17.7 \
```

Comment:
For me, this line still does not work. Elasticsearch crashes when I try to run the test.
In the line above you still mention "Elasticsearch v6 instance" this should refer to version 7.

Reply:
How strange... Is there no error message? If it's this flaky I don't think much is lost by just removing this bit from the docs

Reply:
After a reboot the line you proposed seems to be working again, although I still do not know why it failed before. I did not get any error message in the terminal even after removing "-d". I have not looked in the docker logs though.